### PR TITLE
Optimize ProfileTaskChannelService

### PR DESCRIPTION
**Pull request by chanjarster**
related issue  https://github.com/apache/skywalking/issues/8221

1. Avoid ProfileTaskChannelService.addProfilingSnapshot throw IllegalStateException(Queue full)
2. Increase ProfileTaskChannelService.snapshotQueue default size from 50 to 4500

### CODE REVIEW

File: docs/en/setup/service-agent/java-agent/configurations.md [hunk 1]
```diff
@@ -1,111 +1,111 @@
...
-`plugin.toolkit.log.transmit_formatted` | Whether or not to transmit logged data as formatted or un-formatted. | `true` |
-`plugin.lettuce.trace_redis_parameters` | If set to true, the parameters of Redis commands would be collected by Lettuce agent.| `false` |
-`plugin.lettuce.redis_parameter_max_length` | If set to positive number and `plugin.lettuce.trace_redis_parameters` is set to `true`, Redis command parameters would be collected and truncated to this length.| `128` |
-`plugin.neo4j.trace_cypher_parameters`|If set to true, the parameters of the cypher would be collected.|`false`|
-`plugin.neo4j.cypher_parameters_max_length`|If set to positive number, the `db.cypher.parameters` would be truncated to this length, otherwise it would be completely saved, which may cause performance problem.|`512`|
-`plugin.neo4j.cypher_body_max_length`|If set to positive number, the `db.statement` would be truncated to this length, otherwise it would be completely saved, which may cause performance problem.|`2048`|
+property key | Description | **System Environment Variable** | Default
```
Comment (wu-sheng) [inline docs/en/setup/service-agent/java-agent/configurations.md:4]: Hi, @chanjarster This is a good update, but could you submit this through another pull request? As this is irrelevant with this PR fix.

Comment (chanjarster) [reply docs/en/setup/service-agent/java-agent/configurations.md:4]: OK

**Comment by mrproliu** [comment]
Will you also update the default size of the thread snapshot queue?

**Comment by chanjarster** [comment]
> Will you also update the default size of the thread snapshot queue?

What size is suitable?

**Comment by mrproliu** [comment]
> > Will you also update the default size of the thread snapshot queue?
> 
> What size is suitable?

I have left a comment on [there](https://github.com/apache/skywalking/issues/8221#issuecomment-983391368), I think `4500` is enough, WDYT?

**Comment by chanjarster** [comment]
@mrproliu No problem, code updated.

**Review by wu-sheng — APPROVED** [review]

**Review by mrproliu — APPROVED** [review]
Thanks.
